etl/src/main/java/org/pih/hivmigration/etl/sql/ProgramMigrator.groovy

```diff
@@ -0,0 +1,252 @@
+package org.pih.hivmigration.etl.sql
+
+class ProgramMigrator extends SqlMigrator {
+
+    void migrate() {
+        executeMysql("Create staging table to receive program data", '''
+            create table hivmigration_programs_raw (
+                source_patient_id int PRIMARY KEY,
+                starting_health_center int,
+                health_center int,
+                health_center_transfer_date date,
+                treatment_status varchar(30),
+                treatment_status_date date,
+                art_start_date date,
+                regimen_outcome varchar(20),
+                regimen_outcome_date date,
+                min_visit_date date,
+                max_visit_date date,
+                enrollment_date date,
+                outcome_date date,
+                outcome varchar(255)
+            );
+        ''')
+
+        executeMysql("Create intermediate staging table", '''
+            create table hivmigration_programs (
+                patient_program_id int PRIMARY KEY AUTO_INCREMENT,
+                source_patient_id int,
+                location_id int,
+                enrollment_date date,
+                art_start_date date,
+                outcome_date date,
+                outcome varchar(255)
+            );
+        ''')
+
+        executeMysql("Create outcome lookup table", '''
+            create table hivmigration_outcome (
+              id int PRIMARY KEY AUTO_INCREMENT,
+            	outcome varchar(255),
+            	outcome_concept_id int
+            );
+            
+            SET @outcome_died = (select concept_id from concept where uuid = '3cdd446a-26fe-102b-80cb-0017a47871b2');
+            SET @outcome_lost_to_followup = (select concept_id from concept where uuid = '3ceb0ed8-26fe-102b-80cb-0017a47871b2');
+            SET @outcome_transferred_out = (select concept_id from concept where uuid = '3cdd5c02-26fe-102b-80cb-0017a47871b2');
+            SET @outcome_treatment_stopped = (select concept_id from concept where uuid = '3cdc0d7a-26fe-102b-80cb-0017a47871b2');
+            
+            insert into hivmigration_outcome(outcome, outcome_concept_id) values("DIED", @outcome_died);
+            insert into hivmigration_outcome(outcome, outcome_concept_id) values("LOST", @outcome_lost_to_followup);
+            insert into hivmigration_outcome(outcome, outcome_concept_id) values("TREATMENT_STOPPED", @outcome_treatment_stopped);
+            insert into hivmigration_outcome(outcome, outcome_concept_id) values("TRANSFERRED_OUT", @outcome_transferred_out);
+            insert into hivmigration_outcome(outcome, outcome_concept_id) values("TRANSFERRED_INTERNALLY", @outcome_transferred_out);
+        ''')
+
```

Comment:
I'd remove the above lines in favor of consolidating with the update code around line 104